ee/dtr/user/manage-images/sign-images/trust-with-remote-ucp.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Using Docker Content Trust with a Remote UCP Cluster
+description: Learn how to use a single DTR's trust data with remote UCPs.
+keywords: registry, sign, trust, notary
+---
+
+For more advanced deployments, you may want to share one Docker Trusted Registry
+across multiple Universal Control Planes. However, customers wanting to adopt
+this model, alongside the [Only Run Signed
+Images](../.../../ucp/admin/configure/run-only-the-images-you-trust.md) feature
+of UCP, run into problems as each Universal Control Plane operates an
+independent set of users.
+
+Docker Content Trust with a Remote UCP gets around this problem, as User's from
+a remote UCP are able to sign images in the central DTR, and still apply runtime
+enforcement.
+
+In the following example we will connect DTR to 1 remote UCP cluster, however
+this process can be repeated over and over with more remote UCPs. 
+
+![](../../../images/remoteucp-graphic.png)
+
+> Before attempting this guide, please familiarize yourself with [Docker Content
+> Trust](engine/security/trust/content_trust/#signing-images-with-docker-content-trust)
+> and [Only Run Signed
+> Images](../.../../ucp/admin/configure/run-only-the-images-you-trust.md) on a
+> single UCP. A lot of the concepts within this guide may be new without that
+> background. 
+
+## Prerequisites
+
+- Cluster 1, running UCP 3.0.x or higher, with a DTR 2.5.x or higher deployed
```

Comment:
@ollypom Just looking at this quickly - I wonder if there's a good way to provide a graphic for the prerequisites. Thoughts?